deploy_dynatrace.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+source ${BASE_PATH}/utils.sh
+
+echo "Deploying Dynatrace OneAgent Operator"
+wget https://github.com/dynatrace/dynatrace-operator/releases/latest/download/install.sh -O install.sh && sh ./install.sh --api-url "https://${DT_TENANT}/api" --api-token "${DT_API_TOKEN}" --paas-token "${DT_PAAS_TOKEN}" --enable-k8s-monitoring --enable-volume-storage
```

Comment:
This here would be an attack vector. If anyone manages to inject code into `install.sh` that just prints all environment variables and pushes it to another server (happened to codecov), credentials will be leaked.

Reply:
After a short discussion with Florian about this, here is our take on how to solve this (for now):

Download not the `latest` release, but let's specify a release, e.g.:
```
DYNATRACE_OPERATOR_RELEASE="v0.2.1"
wget https://github.com/dynatrace/dynatrace-operator/releases/download/${DYNATRACE_OPERATOR_RELEASE}/install.sh -O install.sh
```

Audit the content of install.sh manually, calculate a hashsum (e.g., `sha256sum install.sh`): `eb48ab22e9b6dac110cfd9f130045b49c9f12b9f3850f76e2657d9071342a1c1`

Verify the sha256sum within the script:
```
echo "eb48ab22e9b6dac110cfd9f130045b49c9f12b9f3850f76e2657d9071342a1c1 install.sh" | sha256sum --check
```

Abort if it fails

And after that, execute `sh ./install.sh --api-url ...`